react-form-controls/hannah-app/src/RegistrationFormControlled.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log({ username }, { password });
```

Comment:
I might look nicer to log just one object instead of 2: `console.log( { username, password } );`.